src/verify.c

```diff
@@ -606,6 +606,8 @@ int verify_main(int argc, char **argv)
 	struct manifest *official_manifest = NULL;
 	int ret;
 	int lock_fd;
+	int retries = 0;
+	int timeout = 5;
```

Comment:
The retries in update.c use a timeout of 10 seconds. Did you want to use 10 here too?